test/expected/pg_dump.out

```diff
@@ -112,30 +112,31 @@ SELECT * FROM test.show_columns('_timescaledb_internal._hyper_1_1_chunk');
  series_bool | boolean          | f
 (6 rows)
 
-SELECT * FROM test.show_indexes('"test_schema"."two_Partitions"');
+-- COLLATE "C" so the output is the same on Unix & Windows
+SELECT * FROM test.show_indexes('"test_schema"."two_Partitions"') ORDER BY "Index"::TEXT COLLATE "C";
```

Comment:
Ah, so, I think a better way to do this is to modify the `show_indexes` function directly in `test/sql/utils/testsupport.sql`. Instead of `ORDER BY c.oid` at the end of the function, do `ORDER BY c.relname` or `ORDER BY c.oid::regclass` or similar.